blocks/editable/index.js

```diff
@@ -159,8 +159,8 @@ export default class Editable extends Component {
 	}
 
 	registerCustomFormatters() {
-		forEach( this.props.formatters, ( formatter ) => {
-			this.editor.formatter.register( formatter.format, formatter.formatter );
+		forEach( this.props.initialFormatters, ( formatter ) => {
```

Comment:
Minor: Actually, I'd prefer leaving the prop as "formatters", coupled with the `console.error` but I'm fine with both